R/tm_a_regression.R

```diff
@@ -215,57 +215,55 @@ ui_a_regression <- function(id, ...) {
         )
       )
     ),
-    forms = teal::get_rcode_ui(ns("rcode")),
+    forms = teal.widgets::verbatim_popup_ui(ns("rcode"), "Show R code"),
     pre_output = args$pre_output,
     post_output = args$post_output
   )
 }
 
 
 srv_a_regression <- function(id,
-                             datasets,
+                             data,
```

Comment:
Vignette gives me this:

![image](https://user-images.githubusercontent.com/15201933/187472475-c7af1b7e-5a03-4fa8-bc54-14c4777b3856.png)

Reply:
This is not obvious one, as we are using functions instead of reactive's ... will check it

Reply:
fixed